datastore/providers/analyticdb_datastore.py

```diff
@@ -252,7 +253,7 @@ def create_results(data):
                             QueryResult(query=query.query, results=results)
                         )
                 except Exception as e:
-                    print("error:", e)
+                    logger.exception("error:", e)
```

Comment:
Doesn't logger.exception already logs the stack trace along with the error message?
So: `logger.exception("<descriptive message>")` should be sufficient, this applies to all cases in the PR

Reply:
Pushed new changes, replaced exception with error. Exception is verbose stack trace and it is logging PII. Yes this is tested.